content/_partners/faq._en.md

```diff
@@ -19,52 +19,48 @@ general_accordion:
+            Login.gov is not a standalone federal agency. We are a program of the [General Services Administration](https://www.gsa.gov/) (GSA), an agency of the U.S. federal government. The program is run by the [Technology Transformation Services](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services) (TTS), a group that leads the digital transformation of the federal government by helping agencies build, buy, and share technology that allows them to provide more accessible, efficient, and effective products and services for the American people.
     -
         id: a-5
         title: Does Login.gov partner with state and local governments?
         content: >-
-            Yes, Login.gov partners with state and local governments. State and local governments need simple and secure solutions to help the public access federally funded services and resources. With this partnership, those agencies can leverage Login.gov to create a seamless and secure sign in experience for the public to access these resources.  [Learn more about the path to partnership](/partners/state-and-local/){:class="usa-nav_link caret"}
+            Yes, Login.gov partners with state, local, and territory governments. These government entities need simple and secure solutions to help the public access services and resources, and with this partnership, they can leverage Login.gov to create a seamless and secure sign-in experience for the public to access these services and resources. [Learn more about the path to partnership](/partners/state-and-local/){:class="usa-nav_link caret"}
     -
         id: a-6
         title: How do we partner with Login.gov?
         content: >-
-            [Contact an account manager to get started](/partners/contact/){:class="external-link"}. We’ll work with you to understand and capture your needs and requirements at a high level. Together, we’ll decide whether Login.gov makes sense for your particular business and use cases. If we decide to move forward, the next step is to sign an Interagency Agreement (IAA). This signals a mutual commitment which allows us to commit further resources to technical discovery and integration and migration planning.
+            [Contact an account manager to get started](/partners/contact/){:class="external-link"}. We’ll work with you to understand and capture your needs and requirements at a high level. Together, we’ll decide whether Login.gov makes sense for your particular agency and use case. If we decide to move forward, the next step is to sign an [Interagency Agreement (IAA)](/partners/get-started/#interagency-agreement-iaa-process). This signals a mutual commitment which allows us to commit further resources to technical discovery and integration and migration planning.
```

Comment:
This link at the beginning was changed to "Contact our Partnerships Team to get started." according to the content doc.